### PR TITLE
Ivy eye and logspace

### PR DESCRIPTION
**Pull request by SayanJerr**
I have reverted the array test to as is in the current ivy version
for:
#945 
#963 
@djl11

### CODE REVIEW

**Review by 1Doomdie1 — CHANGES_REQUESTED** [review]
Please resolve the conflicts

File: ivy/functional/ivy/linear_algebra.py [hunk 1]
```diff
@@ -714,7 +714,8 @@ def cholesky(
 
      3. Returns a lower-triangular Cholesky factor
 
-     >>> x = ivy.Container(a = ivy.array([[3., -1.], [-1., 3.]]) , b = ivy.array([[2., 1.], [1., 1.]]))
+     >>> x = ivy.Container(a = ivy.array([[3., -1.], [-1., 3.]]) ,
```
Comment (bicycleman15) [inline ivy/functional/ivy/linear_algebra.py:717]: you would need a `\` character at the end of the line for this to pass docstring tests

File: ivy/functional/ivy/creation.py [hunk 2]
```diff
@@ -382,22 +408,31 @@ def eye(
...
     -------
     ret
-        device on which to place the created array. Default: None.
+        a two-dimensional array with ones on the k h diagonal and zeros elsewhere.
 
+    Examples
+    --------
+    >>> y = ivy.eye(2)
+    >>> print(y)
+    ivy.array([[1, 0],[0, 1]])
```
Comment (bicycleman15) [inline ivy/functional/ivy/creation.py:435]: I think the examples are still not exhaustive. Please refer [here](https://lets-unify.ai/ivy/contributing/13_docstring_examples.html)

File: ivy/functional/ivy/creation.py [hunk 3]
```diff
@@ -382,22 +408,31 @@ def eye(
...
+        a negative value to a lower diagonal,
+        and 0 to the main diagonal. Default: 0.
     dtype
-        output array data type. If dtype is None, the output array data type must be the
-        default floating-point data type. Default: None.
+        output array data type.
+        If dtype is None, the output array data type must be the default floating-
+        point data type. Default: None.
     device
          device on which to place the created array.
```
Comment (bicycleman15) [inline ivy/functional/ivy/creation.py:424]: Please update the docstring according to [this](https://lets-unify.ai/ivy/contributing/12_docstrings.html)

File: ivy/functional/ivy/creation.py [hunk 4]
```diff
@@ -669,5 +713,10 @@ def logspace(
     ret
         Tensor of evenly-spaced values.
 
+    Examples
+    --------
+    >>> y = ivy.logspace(2,3,num=4)
+    >>> print(y)
+    ivy.array([ 100.        ,  215.443469  ,  464.15888336, 1000.        ])
```
Comment (bicycleman15) [inline ivy/functional/ivy/creation.py:720]: same thing as `eye`, not enough examples. Kindly refer to the contribution guidelines.

**Comment by bicycleman15** [comment]
I would suggest you to make a new PR perhaps, this one has become very messy and difficult to review. Let me know if you do make a new PR, I will then close this one 🙂

**Comment by juliagsy** [comment]
Hey there! As the PR is inactive for more than 7 days, it will be closed in 24 hours if there are no further response. Thank you!

**Comment by juliagsy** [comment]
Closing due to inactivity

**Pull request closed without merging**